tests/DecimalTest.php

```diff
@@ -67,25 +55,72 @@ public function testCreate($value, string $expected): void
      */
     public function baseProvider(): array
     {
+        $objectWithToStringMethod = new class
+        {
+            /**
+             * @return string
+             */
+            public function __toString(): string
+            {
+                return '12.12';
+            }
+        };
+
         return [
+            [1.1, '1.1'],
+            [-23, '-23'],
             [50, '50'],
             [-25000, '-25000'],
-            [0.00001, '0.000010'], // !
-            [-0.000003, '-0.0000030'], // !
+            [0.00001, '0.00001'],
```

Comment:
Fixed in line https://github.com/spryker/decimal-object/pull/6/files#diff-7a6dc60c53184be6293c42af8fd46fd3R789